product/routes/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const controllers = require('#productCtrl')
+
+const router = express.Router()
+
+router
+  .get(
+    '/helloworld',
+    controllers.helloworld.helloworld
+  )
+  .get(
+    '/question/list',
+    controllers.question.list
+  )
+  .get(
```

Comment:
Nice. Right now you are picking up a random question. We will improve this request a bit more in the next iteration